routers/projectsRouter.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router = express.Router();
+const Projects = require('../data/helpers/projectModel');
+
+
+router.get('/', (req, res) => {
+  Projects.get()
+    .then(projs => {
+      console.log("Projects router: projs", projs);
+      res.status(200).json(projs);
+    })
+
+})
+
+router.get('/:id', validateResId, (req, res) => {
+  Projects.get(req.params.id)
+    .then(proj => {
+      if (!proj) {
+        return res.status(404).json({ message: "There's no project matching that id in the database" })
+      }
+      console.log("Projects router: projs", proj);
+      res.status(200).json(proj);
+    })
+
+})
+
+router.post('/', (req, res) => {
+  Projects.insert(req.body)
+    .then(newProject => {
+      res.status(201).json(newProject);
+    })
+    .catch(err => { res.status(500).json({ errorMessage: "Something went wrong with your post" }) })
+})
```

Comment:
Don't froget to validate the `req.body` to make sure it has the required fields!

Reply:
This code actualy looks a bit different from our 1:1, so I think you've updated it a bit since then 👍